src/topology/local_extr.lean

```diff
@@ -252,6 +252,24 @@ lemma is_local_extr.comp_continuous_on [topological_space δ] {s : set δ} (g :
 hf.elim (λ hf, (hf.comp_continuous_on hg hb).is_extr)
   (λ hf, (is_local_max.comp_continuous_on hf hg hb).is_extr)
 
+lemma is_local_min_on.comp_continuous_on [topological_space δ] {t : set α} {s : set δ} {g : δ → α} {b : δ}
+  (hf : is_local_min_on f t (g b)) (hst : s ⊆ g ⁻¹' t) (hg : continuous_on g s) (hb : b ∈ s) :
+  is_local_min_on (f ∘ g) s b :=
+let h₁ := (continuous_within_at.tendsto_nhds_within_image (hg b hb)) in
```

Comment:
What is the point of those `let` here and in the next proof? Is there an elaboration issue?

Reply:
No, there is no elaboration issue, it was just to break up the line -- is this style frowned upon...?

Reply:
It is very unusual. I'd say we should keep our usual style since this variant brings no benefit.

Reply:
Fair enough, I'll remove the lets.